numba/ir_utils.py

```diff
@@ -1822,6 +1822,14 @@ def raise_on_unsupported_feature(func_ir, typemap):
     unsupported features.
     """
     gdb_calls = [] # accumulate calls to gdb/gdb_init
+    
+    for arg_name in func_ir.arg_names:
+        if arg_name in typemap and \
+           isinstance(typemap[arg_name], types.containers.UniTuple) and \
+           (typemap[arg_name].count > 1000):
```

Comment:
Perhaps leave a comment here for developers noting the issue from which this arose and that 1000 is entirely arbitrary.